src/latexify/codegen/latex_test.py

```diff
@@ -2,7 +2,8 @@
 
 from __future__ import annotations
 
-from latexify.codegen.latex import Latex  # Ignores [22-imports] for convenience.
+from latexify.codegen.latex import \
+    Latex  # Ignores [22-imports] for convenience.
```

Comment:
some mistakes?
```suggestion
from latexify.codegen.latex import Latex  # Ignores [22-imports] for convenience.
```

Reply:
oh yeah black keeps on formatting it that way for some reason, I've had to `--assume-unchanged` but must've missed it here